crates/aptos/src/move_tool/mod.rs

```diff
@@ -64,6 +68,7 @@ impl MoveTool {
             MoveTool::Run(tool) => tool.execute_serialized().await,
             MoveTool::Test(tool) => tool.execute_serialized().await,
             MoveTool::Prove(tool) => tool.execute_serialized().await,
+            MoveTool::RunTransactionalTest(tool) => tool.execute_serialized().await,
```

Comment:
You can also use `execute_serialized_success()` and then make the return type `CliTypedResult<()>` below.